src/kits/base/omni_base_node.cpp

```diff
@@ -137,15 +137,15 @@ int main(int argc, char ** argv) {
     ROS_INFO("Found and successfully read 'families' parameter");
   } else {
     ROS_INFO("Could not find/read 'families' parameter; defaulting to 'HEBI'");
-    families = {"HEBI"};
+    families = {"Rosie"};
   }
 
   std::vector<std::string> names;
   if (node.hasParam("names") && node.getParam("names", names)) {
     ROS_INFO("Found and successfully read 'names' parameter");
   } else {
     ROS_INFO("Could not find/read 'names' parameter; defaulting to 'Wheel1', 'Wheel2', and 'Wheel3' ");
-    names = {"Wheel1", "Wheel2", "Wheel3"};
+    names = {"_Wheel1", "_Wheel2", "_Wheel3"};
```

Comment:
This isn't a huge deal, since this isn't consistent yet, but this changes the default repo behavior and may cause issues for customers updating the code.  We might want to leave the family/name as is for now, and make a single deliberate change across all examples.